CotEditor/Storyboards/mul.lproj/Main.xcstrings

```diff
@@ -9389,6 +9971,12 @@
             "value" : "Converter texto alfanumérico para metade da largura"
           }
         },
+        "ru" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Преобразовать буквенно-цифровой текст в половину ширины"
```

Comment:
Could you also update this label along with “Convert to Full Width” and “Convert to Half Width”?